Makefile

```diff
@@ -43,11 +43,12 @@ OFFICIAL_RELEASE ?= true
 PACKAGE_CONFLICTS ?= `grep PACKAGE_CONFLICTS $(KONG_SOURCE_LOCATION)/.requirements | awk -F"=" '{print $$2}'`
 PACKAGE_PROVIDES ?= `grep PACKAGE_PROVIDES $(KONG_SOURCE_LOCATION)/.requirements | awk -F"=" '{print $$2}'`
 PACKAGE_REPLACES ?= `grep PACKAGE_REPLACES $(KONG_SOURCE_LOCATION)/.requirements | awk -F"=" '{print $$2}'`
-DOCKER_RELEASE_REPOSITORY ?= `grep DOCKER_RELEASE_REPOSITORY $(KONG_SOURCE_LOCATION)/.requirements | awk -F"=" '{print $$2}'`
```

Comment:
Did this get tested with kong-ee? I am skeptical this would correctly overwrite with the variable's value in `.requirements` (https://github.com/Kong/kong-ee/blob/c2d854b8dea021ab581588b8e758665a519e4746/.requirements#L9)

Reply:
I couldn't find a way to keep this PR backwards compatible with current Kong CE which doesn't have a value for `DOCKER_RELEASE_REPOSITORY`.

Someway of in order of precedence:
1. use an environment variable
2. use the DOCKER_RELEASE_REPOSITORY value from .requirements
3. some sensible default

Reply:
I think our build infrastructure for the gateway product already presumes that default values pertain to the building of CE. So option 3, a sensible default for the value required for the CE build, makes sense to me.